CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# 2019-02-28 v0.1.4
```

Comment:
The format used in most changelog is:
```
0.6.0 (November 2018)
---------------------
```
i.e. version without ``v`` then Month Year